services/rank_score_feeds/README.md

```
@@ -0,0 +1,198 @@
+# Run pipeline
+orchestrator.run(
+    users_to_create_feeds_for=None,  # All users if None
+    export_new_scores=True,
+    test_mode=False
+)
+```
+
+## Testing
+
+Run tests with:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)